src/draw/drawPolygon.ts

```diff
@@ -0,0 +1,177 @@
+import { IJS } from '../IJS';
+import checkProcessable from '../utils/checkProcessable';
+import { getDefaultColor } from '../utils/getDefaultColor';
+import { getOutputImage } from '../utils/getOutputImage';
+
+import { Point } from './drawLine';
+
+export interface Line {
+  /**
+   * Line slope.
+   */
+  a: number;
+  /**
+   * Line y-intercept.
+   */
+  b: number;
+  /**
+   * Line is vertical
+   */
+  vertical: boolean;
+}
+export interface DrawPolygonOptions {
+  /**
+   * Array of N elements (e.g. R, G, B or G, A), N being the number of channels.
+   *
+   * @default black
+   */
+  color?: number[];
+  /**
```

Comment:
There has been a copy-paste of a comment here.

Reply:
i fixed the comments and extends the type from drawPolylineOptions